examples/http/main.go

```diff
@@ -0,0 +1,36 @@
+// A simple HTTP service.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/ngrok/ngrok-go"
+)
+
+func main() {
+	if err := run(context.Background()); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func run(ctx context.Context) error {
+	_, tun, err := ngrok.ConnectAndStartTunnel(ctx,
+		ngrok.ConnectOptions().
+			WithAuthtoken(os.Getenv("NGROK_AUTHTOKEN")),
+		ngrok.HTTPOptions().
+			WithDomain("golang-demo.ngrok.io"),
```

Comment:
let's remove this and add another example that uses domain selection with a dynamically chosen name. the hello world example should work on the free tier and it needs to be dynamic because any hardcoded name will get grabbed

Reply:
random domain also means the program should print it out